src/test/java/org/openqa/selenium/htmlunit/WebDriverTestCase.java

```diff
@@ -541,19 +542,16 @@ protected WebDriver buildWebDriver() throws IOException {
             throw new RuntimeException("Unexpected BrowserVersion: " + getBrowserVersion());
         }
         if (webDriver_ == null) {
-            final WebClientOptions options = new WebClientOptions();
+            final HtmlUnitDriverOptions driverOptions = new HtmlUnitDriverOptions(getBrowserVersion());
+            
             if (isWebClientCached()) {
-                options.setHistorySizeLimit(0);
+                driverOptions.setCapability(HtmlUnitOption.optHistorySizeLimit, 0);
```

Comment:
This could also be expressed as `HtmlUnitOption.HISTORY_SIZE_LIMIT.getCapabilityKey()`, but that's a bit more verbose and doesn't really enhance the readability of the implementation.